src/components/common/Modal/ModalForm/ModalForm.tsx

```diff
@@ -1,71 +1,147 @@
-        <div className={styles.modal_inputLabel_wrapper}>
-          <label className={styles.modal_label}>Movie url</label>
-          <input
-            className={styles.modal_input}
-            type="text"
-            placeholder={"https://"}
-            style={{ width: "500px" }}
-          />
-        </div>
-        <div className={styles.modal_inputLabel_wrapper}>
-          <label className={styles.modal_label}>Rating</label>
-          <input
-            className={styles.modal_input}
-            type="text"
-            placeholder={"0-10"}
-            style={{ width: "300px" }}
-          />
-        </div>
+      <div className={styles.inputs_wrapper}>
+        <ModalInput
+          register={register}
+          name={"poster_path"}
+          type={"text"}
+          size={"lg"}
+          label={"Movie url"}
+          error={errors.poster_path && "Url is required (https://...jgp/png)"}
+          pattern={/(http(s?):)([/|.\w\s-])*\.(?:jpg|gif|png)/}
+          placeholder={"https://"}
+        />
+        <ModalInput
+          register={register}
+          name={"vote_average"}
+          type={"number"}
+          size={"sm"}
+          label={"Rating"}
+          error={errors.vote_average && "Rating is required (0-10)"}
+          placeholder={"0-10"}
+          valueAsNumber={true}
+        />
       </div>
-      <div className={styles.modal_inputsWrapper}>
-        <div className={styles.modal_inputLabel_wrapper}>
-          <label className={styles.modal_label}>Genre</label>
-          <ModalSelect />
-        </div>
-        <div className={styles.modal_inputLabel_wrapper}>
-          <label className={styles.modal_label}>Runtime</label>
-          <input
-            className={styles.modal_input}
-            type="text"
-            placeholder={"Minutes"}
-            style={{ width: "300px" }}
-          />
-        </div>
+      <div className={styles.inputs_wrapper}>
+        <ModalSelect
+          register={register}
+          setValue={setValue}
+          getValues={getValues}
+          errors={errors}
+        />
+        <ModalInput
+          register={register}
+          name={"runtime"}
+          type={"number"}
+          size={"sm"}
+          label={"Runtime"}
+          error={errors.vote_average && "Runtime is required (minutes)"}
+          placeholder={"Minutes"}
+          valueAsNumber={true}
+        />
       </div>
-      <div className={styles.modal_inputLabel_wrapper}>
-        <label className={styles.modal_label}>Overview</label>
-        <textarea
-          className={styles.modal_textarea}
-          placeholder={"More details"}
-        ></textarea>
+      <ModalTextarea register={register} errors={errors} />
+      <div className={styles.modal_footer}>
+        <Button onClick={() => reset()} btnType={"btn_dark"}>
+          Reset
+        </Button>
+        <Button onClick={handleSubmit(onSubmit)} btnType={"btn_danger"}>
```

Comment:
`onSubmit` event should be defined as form attribute

Reply:
What is the difference between form and button defining?

Reply:
Read [this](https://javascript.info/forms-submit), try to click enter with your form, then give me the answer what's the difference?

Reply:
1. validate the form before sending
2. pressing `Enter` button triggers `submit` event

Is it correct if I add the `type="submit"` attribute to the button than `onSubmit` to the form